node/src/components/storage.rs

```diff
@@ -1283,6 +1283,13 @@ impl Storage {
             .map(|sequence| sequence.high())
     }
 
+    /// Retrieves the height of the lowest complete block (if any).
+    pub(crate) fn lowest_complete_block_height(&self) -> Option<u64> {
+        self.completed_blocks
+            .highest_sequence()
+            .map(|sequence| sequence.low())
+    }
```

Comment:
This is misleading. The function actually returns the height of the lowest block in the highest completed blocks sequence. However, it seems that this is the behavior we want in the use case presented in the PR. I suggest we rename this to something else or simply use `storage::get_available_block_range`. Same applies for `read_lowest_complete_block_header`.

Reply:
Updated here: https://github.com/casper-network/casper-node/pull/3813/commits/0b3421b2212b5e61d8de5593fa3b4b3dfc5800f3